docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md

```diff
@@ -174,7 +187,7 @@ Azure AD Inventory Source.
 | `secret_key` | String | Yes |  | Provide the Application Client Secret Value you created in Azure. | modifiable
 | `application_id` | String | Yes |  | Provide the Application (client) ID you got after you registered (created) the Azure Application. | modifiable |
 | `supported_apis` | Array of strings | Yes |  | Define one or more of the available APIs to collect: Devices, and Users. For example, for both you'd use: ["Devices","Users"] | modifiable |
-
+| `userSignInActivity` | Boolean | No | False | Select the checkbox to include signInActivity data for user | modifiable |
 Microsoft Azure AD Inventory Source JSON example:
```

Comment:
Add a space between line 190 and 191. Add a H3 title before line 191 as JSON example.